pkg/cluster/cluster_info_scraper.go

```diff
@@ -205,17 +205,14 @@ func (s *ClusterScraper) GetRunningAndReadyPodsOnNode(node *api.Node) []*api.Pod
 	return util.GetReadyPods(nodeRunningPodsList)
 }
 
-func (s *ClusterScraper) GetRunningAndReadyPodsOnNodes(nodeList []*api.Node) []*api.Pod {
-	pods := []*api.Pod{}
-	for _, node := range nodeList {
-		nodeRunningPodsList, err := s.findRunningPodsOnNode(node.Name)
-		if err != nil {
-			glog.Errorf("Failed to find running pods in %s", node.Name)
-			continue
-		}
-		pods = append(pods, nodeRunningPodsList...)
+func (s *ClusterScraper) GetPendingPodsOnNode(node *api.Node) []*api.Pod {
+	nodePendingPodsList, err := s.findPodsOnNode(node.Name, api.PodPending)
+	if err != nil {
+		glog.Errorf("Failed to find running pods in %s", node.Name)
```

Comment:
"Failed to find running pods" -> "Failed to find pending pods"

Reply:
Fixed